### PR TITLE
Running imagec in /tmp so it can create files

### LINKED ISSUE #1954: Failure in the image tests
We saw this failure on one build yesterday. 
```

16:04:30.980	INFO	Starting process:
/drone/src/github.com/vmware/vic/bin/imagec -standalone -reference photon -logfile foo.log	
16:04:30.987	INFO	Waiting for process to complete.	
16:04:53.365	INFO	Process completed.	
16:04:53.365	INFO	${result} = <result object with rc 1>
```

### PR DESCRIPTION
**Pull request by hmahmood**
Fixes #1954

### CODE REVIEW

File: tests/test-cases/Group2-Imagec/2-10-Verify-Logfile.robot [hunk 1]
```diff
@@ -3,8 +3,8 @@ Resource  ../../resources/Util.robot
 
 *** Test Cases ***
 Test
-    ${result}=  Run Process  ${bin-dir}/imagec -standalone -reference photon -logfile foo.log  shell=True  cwd=/
+    ${result}=  Run Process  ${bin-dir}/imagec -standalone -reference photon -logfile foo.log  shell=True  cwd=/tmp
```
Comment (caglar10ur) [inline tests/test-cases/Group2-Imagec/2-10-Verify-Logfile.robot]: imagec has a destination parameter so I suggest using that instead of doing this

Comment (hmahmood) [inline tests/test-cases/Group2-Imagec/2-10-Verify-Logfile.robot]: The `-destination` parameter does not work with the log file:
```
:/$ ~/go/src/github.com/vmware/vic/bin/imagec -destination /tmp
```

Comment (caglar10ur) [inline tests/test-cases/Group2-Imagec/2-10-Verify-Logfile.robot]: There is also logfile parameter :)

**Comment by fdawg4l** [comment]
LGTM

**Comment by caglar10ur** [comment]
LGTM :sheep: :it:

**Comment by hmahmood** [comment]
♻️

**Pull request merged**